### PR TITLE
Add tests to gh CI

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
We have few tests but as we write more, it would be good to run them in CI so that they always pass in main.

# Changes
- Added a github action to run tests.
- Added a github action to run lints, albeit ignoring candid and tests.

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -54,3 +54,16 @@ jobs:
...
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
+      - name: Test
```
Comment (peterpeterparker) [inline .github/workflows/checks.yml:68]: `npm run lint` too?

Comment (bitdivine) [reply .github/workflows/checks.yml:68]: ^ Lint fails, otherwise I would have!

**Review by peterpeterparker — APPROVED** [review]
All good as discussed on the chat

File: .github/workflows/checks.yml (hunk 1 above)
Comment (peterpeterparker) [reply .github/workflows/checks.yml:68]: tests ignored for linter in https://github.com/dfinity/nns-js/pull/18

**Pull request merged**